pythonx/jedi_vim.py

```diff
@@ -1027,11 +1030,15 @@ def do_rename(replace, orig=None):
         vim.current.buffer[r.line - 1] = (r_line[:r.column] + replace +
                                           r_line[r.column + len(orig):])
 
+    vim_command('w')
+    vim.current.buffer = saved_buffer
+    vim.current.window.cursor = saved_cursor
     # Restore previous tab and window.
     vim_command('tabnext {0:d}'.format(saved_tab))
     vim_command('{0:d}wincmd w'.format(saved_win))
 
     if len(buffers) > 1:
+        usages()
```

Comment:
Why is this necessary?

Reply:
Was following blueyed's suggestion here: https://github.com/davidhalter/jedi-vim/issues/1017#issuecomment-647227001